README.md

```diff
@@ -1,5 +1,23 @@
 # ffd-info-exchange
 
+## Local installation
+
+This app is designed to run on Python 3.4+. `pyvenv` can manage the dependencies installed with `pip`. With that, you can prepare your development environment by running:
+
+```
+git clone https://github.com/18F/ffd-info-exchange.git
+cd ffd-info-exchange
+pyvenv ffd-env
+source ffd-env/bin/activate
+pip install -r requirements.txt
+createdb ffd-info-exchange
```

Comment:
Good call. I'll see whether there's something preexisting I can leverage/link to.